src/constants.ts

```diff
@@ -3,11 +3,9 @@ import { Duration } from 'aws-cdk-lib';
 export const DEFAULT_STATIC_MAX_AGE = Duration.days(30).toSeconds();
 export const DEFAULT_STATIC_STALE_WHILE_REVALIDATE = Duration.days(1).toSeconds();
 
-export const CACHE_BUCKET_KEY_PREFIX = '_cache';
-
 export const NEXTJS_STATIC_DIR = 'assets';
 export const NEXTJS_BUILD_DIR = '.open-next';
-export const NEXTJS_CACHE_DIR = 'cache';
+export const NEXTJS_CACHE_DIR = '_cache';
```

Comment:
This should not be changed for the same reason as my earlier comment.

Reply:
yep, sorry, it's getting late here, I'm 100% on the same page now. 

I'll keep both, just update the S3 deployed directory to be `_cache`

Reply:
haha, no worries. thank you!